downstream/modules/platform/con-pod-specification-mods.adoc

```diff
@@ -83,15 +83,15 @@ A pod definition can specify the basis of a name with the `generateName` attribu
 | `env:` | Environment variables pass necessary values to each container.
 | `image:` | Each container in the pod is instantiated from its own Docker-formatted container image.
 | `ports:` | The container can bind to ports made available on the pod’s IP.
-| `resources` | When you specify a Pod, you can optionally describe how much of each resource a container needs.
+| `resources:` | When you specify a pod, you can optionally describe how much of each resource a container needs.
 The most common resources to specify are CPU and memory (RAM).
 Other resources are available.
-| `securityContext` | OpenShift Online defines a security context for containers that specifies whether they are permitted to run as privileged containers, run as a user of their choice, and more.
-The default context is very restrictive but administrators can modify this as required.
+| `securityContext:` | OpenShift Online defines a security context for containers that specifies whether they are permitted to run as privileged containers, run as a user of their choice, and more.
+The default context is very restrictive but administrators can change this as required.
 | `volumeMounts:` | The container specifies where external storage volumes should be mounted within the container.
 In this case, there is a volume for storing the registry’s data, and one for access to credentials the registry needs for making requests against the OpenShift Online API.
 | `ImagePullSecrets` | A pod can contain one or more containers, which must be pulled from some registry.
-If containers come from registries that require authentication, you can provide a list of `ImagePullSecrets` that refer to `ImagePullSecrets` present in the namespace.
+If containers come from registries that require authentication, you can give a list of `ImagePullSecrets:` that refer to `ImagePullSecrets` present in the namespace.
 Having these specified enables {OCP} to authenticate with the container registry when pulling the image.
 For further information, see link:https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/[Managing resource containers] in the kubernetes documentation.
```

Comment:
Should Kubernetes be capitalized in "in the kubernetes documentation"?

Reply:
You are correct, I've fixed it now :)